### DISCUSSION TITLE
Searching for notes with boolean label

### DISCUSSION THREAD

**Discussion by ecating** [TriliumNext/Trilium#2062] [Q&A]
I've added a label to my notes, #label:Handled=promoted,single,boolean, and I want to search for all my notes where the value is FALSE.  I've tried doing 
Search String: #label:Handled = x
where for 'x' I have tried False, FALSE, 'FALSE', 'False', 0, boolean(0), etc...
What is the syntax for a boolean value FALSE in the Trilium search?

Thanks!

**Comment by zadam** [comment]
Hi, sorry for the late response. False is represented by label not being there, this can be searched by `#!handled`.

For search don't use "label:" prefix. This prefix is used only for the promoted attribute definition (itself a label), not for the actual label usage.
